src/routes/Settings/SettingsIndexView.js

```diff
@@ -35,123 +162,48 @@ class SettingsIndexView extends PureComponent<Props, *> {
     const derivationPath = mnemonicOptions ? mnemonicOptions.derivationPath : null
     const passphrase = mnemonicOptions ? mnemonicOptions.passphrase : null
 
+    const SettingsCards = getSettingsCardProperties({
+      localCurrencyCode,
+      defaultGasPrice,
+      systemLanguageCode,
+      hasPinCode,
+      walletName,
+      networkName,
+      isFullMnemonic,
+      derivationPath,
+      passphrase,
+    }).filter((elementProps) => {
```

Comment:
Most of the times users would try to find settings by non-trivial queries. For example, users can look for "language" using "locale" or "translation".

For those cases consider to add smth like `searchTags` field, empty and ignored by default. We will fill it with keywords as soon as we get our hands on real-world data.

Reply:
Meh...